### ISSUE TITLE
[feature request]  Custom allowed features 

### ISSUE THREAD

**Issue by n8v8R** [snyderp/web-api-manager#90]
Basically as a mirror to  `Custom blocked features` it would be handy to have `Custom allowed features` which would assist in fine graining the extention, particularly when it comes to non-standards

**Comment by jawz101** [comment]
agreed.  If I have a default block template and I can just permit 1 or 2 things for specific site I could just create a permissive and just check the boxes to allow a few things which would overwrite the global denial for those few things the global blocks.

**Comment by n8v8R** [comment]
@jawz101 Did you read/see

> This project is not currently maintained. I no longer have the time to keep it supported. 

?

Too bad that is really. I liked very much the direction this extention was moving...

**Comment by jawz101** [comment]
Well, I'm still going to use it as long as it works.  There are sooo many standards he incorporated into the extension its too valuable to stop using.

**Comment by jawz101** [comment]
Yeah.  He put a lot of work into it.  It stinks to see it stagnate.
